python/tk_multi_publish2/processing/plugin_manager.py

```diff
@@ -164,7 +164,7 @@ def _collect(self, collect_current_scene, paths=None):
                     "collector",
                     "process_file",
                     parent_item=self._root_item,
-                    path=path
+                    path=path.decode("utf-8")
```

Comment:
the general approach that we have taken generally in toolkit is to try to intercept any unicode (usually coming from PySide) and convert it into utf-8 as early on as possible. This `path` value has been passed around for a bit before it reaches this point. I suggest tracing back to find the source of the unicode and see if we can perform the conversion there instead. We found that this approach generally reduces complexity!